packages/amplify_analytics_pinpoint/lib/amplify_analytics_pinpoint.dart

```diff
@@ -46,7 +46,7 @@ class AmplifyAnalyticsPinpoint extends AnalyticsPluginInterface {
     return _instance.addPlugin();
   }
 
-  Future<void> recordEvent({@required AnalyticsEvent event}) async {
```

Comment:
`import 'package:flutter/cupertino.dart';` can be removed with the removal of `@required`

Reply:
thanks, removed